oaknational/Oak-Web-Application
# feat: add legacy curriculum downloads

## Description

Music year: 1956

- Create new module for Curriculum Downloads
- Create page for previous (or legacy) curriculum document downloads
- Create fixture to store previous curriculum download data

## How to test

1. Go to https://deploy-preview-2116--oak-web-application.netlify.thenational.academy/teachers/curriculum/previous-downloads
2. You should that the `EYFS` tab is selected by default.
3. Click on the tab `KS1`
4. You should see the tab updated to show relevant downloads.
5. Click on "Music"
6. Fill in the "Your details" section.
7. Click "Download PDF".
8. The Music curriculum document PDF should download.

## Screenshots

How it should now look:
<img width="1091" alt="Screenshot 2024-02-02 at 11 07 48" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/0f8f2bcf-619d-4c50-bb36-56903d97fcc7">
<img width="1091" alt="Screenshot 2024-02-02 at 11 07 58" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/88628b03-873a-4e2f-8e0c-3cb9a0d542a3">
<img width="1091" alt="Screenshot 2024-02-02 at 11 08 07" src="https://github.com/oaknational/Oak-Web-Application/assets/421186/f9eb1108-0593-4373-b0ba-a549802b2d6e">


## Checklist
